src/inspect_ai/_view/www/src/client/api/index.ts

```diff
@@ -36,16 +35,25 @@ const resolveApi = (): ClientAPI => {
     const urlParams = new URLSearchParams(window.location.search);
     const log_file = urlParams.get("log_file");
     const log_dir = urlParams.get("log_dir");
+    const server_list = urlParams.get("server_list") === "true";
```

Comment:
I think perhaps we should call this something like:

`inspect_server`

We could then below leave the existing logic alone and just check for:

```javascript
const forceBrowserApi = urlParams.get("inspect_server") === "true";

if (forceBrowserApi) {
   return clientApi(createBrowserApi({ server_log_dir: log_dir === null ? undefined : log_dir }), resolved_log_file);
} elseif (log_file !== null || log_dir !== null) {
... 
}
```

Same basic idea, really, but I think it is better that passing the param always forces that client (it would be odd to pass that param with no log_dir and not get the browser api).

Reply:
Good point! Done